theme/base-2016/source/bower.json

```diff
@@ -9,9 +9,9 @@
   "license": "MIT",
   "homepage": "http://foundation.zurb.com",
   "dependencies": {
-    "foundation-sites": "~6.2.3",
+    "foundation-sites": "~6.2.4-rc2",
     "motion-ui": "~1.2.2",
-    "what-input": "~3.0.0",
+    "what-input": "~4.0.1",
     "magnific-popup": "^1.1.0",
     "jquery": "~2.2.4"
```

Comment:
@GawainLynch left-pad has been superseded by west-pad. Assuming developers faced a certain direction led to some pretty nasty bugs.